### DISCUSSION TITLE
How do I embed custom fonts

### DISCUSSION THREAD

**Discussion by tottiMG** [Bodmer/TFT_eSPI#2061] [Q&A - Fonts]
Hello,

Somehow I'm just too stupid for that.

How do I integrate the fonts that are in the Fonts/Custom folder?

Somehow I can't get it.

Then I still have the problem that I don't understand how I can change existing fonts with the letters ä, Ä, ö, Ö, ü, Ü

thanks in advance

**Comment by Bodmer** [comment]
I suggest you keep the custom fonts local to the sketch as that is the simplest approach.

See example here:
https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Generic/Local_Custom_Fonts

The header of that example provides instructions:
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/Generic/Local_Custom_Fonts/Local_Custom_Fonts.ino#L4-L30
